crates/wast/README.md

```diff
@@ -29,6 +29,7 @@ types to parse anything that looks like a WebAssembly s-expression.
 
 * Need to parse a `*.wat` file?
 * Need to parse a `*.wast` file?
+* Need to parse a `*.cat` file?
```

Comment:
Could we possibly use the `*.wat` extension instead of creating a new one? I was originally under the impression that components were "just another kind of wasm" so if they still have `*.wasm` as the binary format extension then it seems like we'd still want to use `*.wat` as the text extension. This doesn't really affect this PR too much, mainly just the documentation, but I'd ideally like to avoid components feeling like a separate world if we can.